assets/model_monitoring/components/src/generation_safety_quality/annotation_compute_metrics/run.py

```diff
@@ -153,7 +183,23 @@ def run():
                     metadata_schema,
                 )
             aggregated_metrics_df = aggregated_metrics_df.union(threshold_row)
-    save_spark_df_as_mltable(aggregated_metrics_df, args.signal_metrics)
+        if average_score_metric_name in input_metric_names:
+            average_metric_score = _calculate_average_metric_score(
+                histogram_df, full_per_instance_score_metric_name)
+            metric_df = spark.createDataFrame(
+                    [
+                        (
+                            "",
+                            average_metric_score,
+                            average_score_metric_name,
+                            "",
```

Comment:
do we have threshold for this average score?

Reply:
let's skip the thresholds for average score for now, I can add them once we get more clarity on this